src/TurbulenceClosures/isopycnal_rotation_tensor_components.jl

```diff
@@ -1,32 +1,59 @@
 # tracer components of the Redi rotation tensor
 
-@inline function isopycnal_rotation_tensor_xz_fcc(i, j, k, grid::AbstractGrid{FT}, buoyancy, tracers) where FT
+"""
+    AbstractTurbulenceClosure
+
+Abstract supertype for turbulence closures.
+"""
+abstract type AbstractIcopycnalModel end
+
+struct SlopeApproximation <: AbstractIcopycnalModel end
+struct SmallSlopeApproximation <: AbstractIcopycnalModel end
```

Comment:
Do you mean add `slope_limiter` as a property of the isopycnal model? I think that's a good idea...

Reply:
Yeah... because only the `isopycnal_model` knows what the slope is; therefore the limited has to be applied there (or we have to export the slopes, but this is maybe more complicated)

Reply:
Also maybe could be `LinearizedRotationTensor`?

Reply:
Names aside, you were thinking something like this?
```
struct SmallSlopeApproximation{L} <: AbstractIcopycnalModel
    slope_limiter :: L
end
```
with `slope_limiter` being `Nothing` as default or a function if we want it on?